virt/virt-4-9-release-notes.adoc

```diff
@@ -66,6 +66,8 @@ High-performance virtual machine templates are now available for xref:virt-guest
 //CNV-14191 FIPS on CNV
 * You can now deploy {VirtProductName} on xref:../installing/installing-fips.adoc#installing-fips[FIPS-enabled clusters].
 
+* The xref:../virt/install/virt-installing-virtctl.adoc#virt-installing-virtctl[`virtctl` client] is now available to download through the `ConsoleCLIDownload` custom resource (CR) for availability even if the cluster is offline.
```

Comment:
I have some thoughts on this, but they're mostly optional:
- I think this might flow better if phrased in the active voice.
- I'd take out "for availability" because you say "is now available" earlier in the sentence.
- I think you should link directly to the module that you added for this feature. It's not appearing in the preview, which is odd. Did you rebase this feature branch against upstream/enterprise-4.9?

Maybe something like this, but with a different xref:
```
* You can now download the xref:../virt/install/virt-installing-virtctl.adoc#virt-installing-virtctl[`virtctl` client] even if the cluster is offline by using the `ConsoleCLIDownload` custom resource (CR).
```